src/apps/content-editor/src/app/views/ItemList/SetRow/ToggleCell/ToggleCell.js

```diff
@@ -15,26 +18,39 @@ export const ToggleCell = memo(function ToggleCell(props) {
       if (options[0].length > 3 || options[1].length > 3) {
         return (
           <span className={cx(props.className, styles.ToggleCell)}>
-            <ToggleButton
-              name={props.name}
+            <ToggleButtonGroup
+              color="secondary"
               value={props.value}
-              offValue="&nbsp;&nbsp;&nbsp;"
-              onValue="&nbsp;&nbsp;&nbsp;"
-              onChange={props.onChange}
-            />
-            {props.field.settings.options[props.value]}
+              size="small"
+              onChange={(e, val) => {
+                if (val !== null) {
+                  props.onChange(val, props.name);
+                }
+              }}
+              onClick={(e) => e.stopPropagation()}
```

Comment:
Lots not do 1 character vars. I would prefer. 

- `err` for error
- `evt` for event